src/utils/engines.ts

```diff
@@ -1,16 +1,14 @@
-import nuxiPkg from '../../package.json'
-
 export async function checkEngines() {
   const satisfies = await import('semver/functions/satisfies.js').then(
     (r) =>
       r.default || (r as any as typeof import('semver/functions/satisfies.js')),
   ) // npm/node-semver#381
   const currentNode = process.versions.node
-  const nodeRange = nuxiPkg.engines.node
+  const nodeRange = '>= 18.0.0'
```

Comment:
could we respect the `engines.node` in the nuxt package, perhaps, to allow for changes here without a nuxi upgrade?

Reply:
Constraints of CLI are not exactly the same as Nuxt Core We might need a version constraint that is only needed for CLI core. 

This also requires the overhead of resolving all possible nuxt packages before every command.